src/payment/strategies/bolt/bolt-app-payment-strategy.ts

```diff
@@ -54,7 +58,14 @@ export default class BoltAppPaymentStrategy implements PaymentStrategy {
             throw new InvalidArgumentError('Unable to submit payment because "payload.payment" argument is not provided.');
         }
 
+        const { isStoreCreditApplied: useStoreCredit } = this._store.getState().checkout.getCheckoutOrThrow();
+
+        if (useStoreCredit !== undefined) {
+            await this._store.dispatch(this._storeCreditActionCreator.applyStoreCredit(useStoreCredit));
+        }
+
         await this._store.dispatch(this._orderActionCreator.submitOrder(order, options));
+        await this._store.dispatch(this._remoteCheckoutActionCreator.initializePayment(payment.methodId, { useStoreCredit }));
```

Comment:
it doesn't seem like remoteCheckout do anything special for Bolt. Can you confirm that store credit works doesn't need this call for Bolt?

<img width="452" alt="Screen Shot 2020-08-19 at 9 47 31 am" src="https://user-images.githubusercontent.com/1621894/90576266-05e59c80-e201-11ea-8b68-172d84e87b88.png">

Reply:
I've removed the `remoteCheckoutActionCreator` it was not needed.